allenact/algorithms/onpolicy_sync/vector_sampled_tasks.py

```diff
@@ -13,27 +13,26 @@
 from typing import (
     Any,
     Callable,
+    Dict,
+    Generator,
+    Iterator,
     List,
     Optional,
     Sequence,
     Set,
     Tuple,
     Union,
-    Dict,
-    Generator,
-    Iterator,
     cast,
 )
 
 import numpy as np
-from gym.spaces.dict import Dict as SpaceDict
-from setproctitle import setproctitle as ptitle
-
 from allenact.base_abstractions.misc import RLStepResult
 from allenact.base_abstractions.task import TaskSampler
 from allenact.utils.misc_utils import partition_sequence
 from allenact.utils.system import get_logger
 from allenact.utils.tensor_utils import tile_images
+from gym.spaces.dict import Dict as SpaceDict
```

Comment:
Same as above. Standard python imports first, third party imports next, allenact imports last.

Reply:
👍 Will revert import changes (see https://github.com/allenai/allenact/pull/339#discussion_r831670937 for details).